module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1171,6 +1171,10 @@ public function getMyHolds($patron)
             // Set expire date if it was included in the response
             $expireDate = isset($hold->requestExpirationDate)
                 ? date_create($hold->requestExpirationDate) : null;
+            //set lastPickup Date if provided, format to j M Y
+            $lastPickup = isset($hold->holdShelfExpirationDate)
+                ? date_format(date_create($hold->holdShelfExpirationDate), "j M Y")
```

Comment:
I think you probably want to use `$this->dateConverter->convertToDisplayDate("Y-m-d H:i", $hold->holdShelfExpirationDate)` here. Note that the format string (my example is "Y-m-d H:i", which may not be correct) is intended to describe the EXISTING format of the date, so that VuFind can parse it and reformat it to the display format specified by config.ini.